osmaxx-py/osmaxx/excerptexport/static/excerptexport/scripts/map.js

```diff
@@ -74,7 +84,7 @@
 
         this._setLocationFilterFromExcerptID = function(ID) {
             var that = this;
-            this.selectedExcerptGeoJson = L.geoJson.ajax("/api/bounding_geometry_from_excerpt/"+ID+"/").on('data:loaded', function(){
+            this.selectedExcerptGeoJson = L.geoJson.ajax("/api/bounding_geometry_from_excerpt/"+ID+"/?format=json").on('data:loaded', function(){
```

Comment:
@hixi Should clients need to specify `format=json`? I'd expect this to be the default if the client doesn't specify any preferred content types. (Or _does_ `L.geoJson.ajax` specify some HTML content types as preferred, thus making it necessary to override that with an URL parameter?)